README.md

```diff
@@ -284,10 +305,15 @@ Before proceeding, ensure that you have installed and configured [aad-pod-identi
     
     If you'll be using Velero to backup multiple clusters with multiple blob containers, it may be desirable to create a unique identity name per cluster rather than the default `velero`.
 
-2. Assign the identity a role:
+2. Assign the identity roles:
 
     ```bash
-    export IDENTITY_ASSIGNMENT_ID="$(az role assignment create --role $AZURE_ROLE --assignee $IDENTITY_CLIENT_ID --scope /subscriptions/$AZURE_SUBSCRIPTION_ID --query id -otsv)"
+    # with custom role
+    az role assignment create --role $AZURE_ROLE --assignee "velero" --scope /subscriptions/$AZURE_SUBSCRIPTION_ID
+    # with predefined roles
+    az role assignment create --role "Contributor" --assignee "velero" --scope /subscriptions/$AZURE_SUBSCRIPTION_ID
+    # not needed when subscriptionId and resourceGroup is set (deprecated)
```

Comment:
Same comment with above, let's remove `# not needed when subscriptionId and resourceGroup is set (deprecated)`